src/driver/sqlx_common.rs

```diff
@@ -1,15 +1,5 @@
 use crate::{DbErr, RuntimeErr};
 
-/// Converts an [sqlx::error] execution error to a [DbErr]
-pub fn sqlx_error_to_exec_err(err: sqlx::Error) -> DbErr {
-    DbErr::Exec(RuntimeErr::SqlxError(err))
-}
-
-/// Converts an [sqlx::error] query error to a [DbErr]
-pub fn sqlx_error_to_query_err(err: sqlx::Error) -> DbErr {
-    DbErr::Query(RuntimeErr::SqlxError(err))
-}
```

Comment:
Personally prefer keeping both method and take a `DbBackend` for converting the error to DB specific error

Reply:
My point in removing these from the common space was, that the implementations where no longer common, but separate for each backend. Hence it has to be in relation to the backend to see how an error has to be read. Since these methods where mostly wrappers before, I didn't see a loss in removing them.